tst/testbugfix/2018-06-28-grplatt.tst

```diff
@@ -0,0 +1,7 @@
+# #2586
```

Comment:
If we ever switch away from GitHub, this reference will be completely inscrutable. Could you perhaps change it to e.g. this?
```
# Verify that ConjugacyClassesSubgroups returns classes with representatives that actually are
# contained in the group (bug #2586 on GitHub)
```
Or if you want to stay brief (as the test actually is quite self-explanatory), perhaps at least
```
# verify fix for bug #2586 on GitHub
```